class-gwiz-gf-code-chest.php

```diff
@@ -382,6 +385,25 @@ public function maybe_unhook_legacy_custom_js( $form ) {
 		remove_filter( 'gform_register_init_scripts', array( $gf_custom_js_instance, 'register_init_script' ), 99 );
 	}
 
+	public function maybe_execute_custom_php( $form ) {
+
+		$custom_php = $this->get_custom_php( $form );
+
+		if ( empty( $custom_php ) ) {
+			error_log( '[Code Chest] No PHP code found for form ' . $form['id'] );
```

Comment:
This will ensure the error gets added to a plugin specific log file:

```suggestion
			$this->log_error( '[Code Chest] No PHP code found for form ' . $form['id'] );
```

Reply:
I have implemented '$this->log_error' instead of 'error_log'